src/Select.jsx

```diff
@@ -388,6 +389,14 @@ export default class Select extends React.Component {
       } else if (isMultipleOrTags(props) && inputValue) {
         // why not use setState?
         this.state.inputValue = this.getInputDOMNode().value = '';
+
+        if (findIndexInValueByLabel(value, inputValue) === -1) {
+          value = [...value, {
+            key: inputValue,
+            label: inputValue,
+          }];
```

Comment:
On the last thing, should call `fireSelect` as well.

Reply:
Done. Nice, thanks